lib/rules/base.js

```diff
@@ -58,7 +63,8 @@ module.exports = class Base {
     this._configStack = [this.config];
 
     this._filePath = options.filePath;
-    this._moduleName = options.moduleName;
+    this[MODULE_ID] = options.moduleId;
```

Comment:
I think we can kill this one, since we (apparently) didn't expose it before.

Reply:
Yep, removed.